Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Scourge Dreadlords

# Changelog:
- Ner'zhul is killed when he tries to kill or imprison his jailors publically.
- The Scourge is shattered when dreadlords kill Ner'zhul.
- Added the Jailor relationship so Ner'zhul sees who is his jailor (need to wait one day to set the relationship.)
- You can't send a poem to the Lich King... probably he won't read it.
- The Scourge gets a warning about summoning the Legion after 10 years of existing, after 15 years dreadlords kill him (reminder that the Scourge created in the year 591)
- Demons and the Scourge consider each other astray as long as Ner'zhul obeys them (so before the year 605 bookmark.)
- Jailors never accept Lich King's faith conversion.
- When all the jailors are dead (somehow), the Scourge stops obeying the Legion.

# Developer changelog:
- `trait_specific.6001` blocked for incapable.
- `on_demon_death` is added.

# Tasks:
- [x] Add on_action for demon death
- [x] Start story in 603 and 605 dates
- [x] Fix dreadlords having empty CoAs
- [x] Ner'zhul is punished when he fails assisination
- [x] When you try to murder or imprison dreadlord, show warning
- [x] Add some game concept about Ner'zhul
- [x] Add jailor relation
- [x] Add Scourge game concept
- [x] Scourge doesnt end on character death
- [x] Story not passed
- [x] Add doctrine icon


## Tests:


# How to test:
Start in the year 603, experiment with dreadlords to test new features.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/735/files?authenticity_token=%2FJPEN%2BdqMesBQq3mwHoFbxomWBdbmWaIQ1vlwjLzQhDl0jNrCpnt4wIaGGXEUxLg%2BPNmSYX%2FbHTQtUqQ4qjBgQ%3D%3D&file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here's the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).
